### ISSUE TITLE
S2 A7 Identifying Objective and Behavioral Statements 

### ISSUE THREAD

**Issue by ajp9e** [castl350/Toddler#95]
On the intro page we list 'In this lesson...' twice.

None of the videos load on the ipad (error message no source video found) 

And when you chose the incorrect answers the 'Incorrect please try again' goes to the next video instead of back to the question

Part 1 of 5 When you chose the incorrect item 'The teacher asks ...' there is a typo in the feedback, should be 'describe' not described

Part 5 of 5 The bubbles for the questions are not aligned. Resizing doesn't help. There is a space between items 2 and 3.

When you 'Review this lesson' it just starts the lesson over again rather than reviewing it

**Comment by castl350** [comment]
Matt and Sarah, can you look at the videos for this. I think this is an activity that we hadn't gotten to because it was added in the line-up later. I'll test it out for flow.

**Comment by castl350** [comment]
I fixed the part 1 of 5 type. For part 5 of 5, nothing we can do about that as it is an LMS issue...Review the Lesson always goes back to the beginning of the lesson. That is a standard. I tested that all of the incorrect choices would go back to the same page to try again and this is the case? Maybe someone else can do a second test before we close this items. The videos played for me...so let's double-check on the iPad and see if there's a difference.
